### PR TITLE
feature/fix-notification-issue-on-loginScreen

### PR DESCRIPTION
**Pull request by Humnaa**
Trello Link : https://trello.com/c/mGcIZ6Hu/59-fix-success-register-notification-issue-on-login-screen

### CODE REVIEW

File: src/components/Header.js [hunk 1]
```diff
@@ -25,12 +26,18 @@ class Header extends Component {
         <div className="collapse navbar-collapse" id="navbarTogglerDemo02">
           <div className="d-lg-flex flex-row-reverse">
             <div className="login-btn">
-              <a
+              {/* <a
```
Comment (ayousaf11) [inline src/components/Header.js]: @Humnaa remove this comment

File: src/containers/Dashboard/Forms/CourseForm.js [hunk 2]
```diff
@@ -16,7 +14,7 @@ class CourseForm extends Component {
       description: ""
     };
     this.handleUserInput = this.handleUserInput.bind(this);
-    this.notificationDOMRef = React.createRef();
+    this.notificationRef =this.props.notificationRef
```
Comment (ayousaf11) [inline src/containers/Dashboard/Forms/CourseForm.js]: @Humnaa add a space after `=`

File: src/containers/Dashboard/Forms/TutorialForm.js [hunk 3]
```diff
@@ -19,7 +17,7 @@ class TutorialForm extends Component {
       courses : []
     };
     this.handleUserInput = this.handleUserInput.bind(this);
-    this.notificationDOMRef = React.createRef();
+    this.notificationRef =this.props.notificationRef;
```
Comment (ayousaf11) [inline src/containers/Dashboard/Forms/TutorialForm.js]: @Humnaa add space after `=`

File: src/components/Header.js (hunk 1 above)
Comment (Humnaa) [reply src/components/Header.js]: I have removed comments

File: src/containers/Dashboard/Forms/CourseForm.js (hunk 2 above)
Comment (Humnaa) [reply src/containers/Dashboard/Forms/CourseForm.js]: done

File: src/containers/Dashboard/Forms/TutorialForm.js (hunk 3 above)
Comment (Humnaa) [reply src/containers/Dashboard/Forms/TutorialForm.js]: ok done

**Review by abdulhaqshah — APPROVED** [review]

**Pull request merged**